packages/grafana-toolkit/src/cli/index.ts

```diff
@@ -233,6 +233,10 @@ export const run = (includeInternalScripts = false) => {
     .option('--commitHash <hashKey>', 'Specify the commit hash')
     .description('Publish to github')
     .action(async (cmd) => {
+      chalk.yellow.bold(`⚠️ This command is deprecated and will be removed . No further support will be provided. ⚠️`);
+      console.log(
+        'We recommend using github actions directly for plugin releasing. You can find an example here:  https://github.com/grafana/plugin-workflows'
```

Comment:
Actually a good question - which ones do we keep up to date? they are starting to drift https://github.com/grafana/plugin-workflows vs https://github.com/grafana/plugin-tools/tree/main/packages/create-plugin/templates/github/ci/.github/workflows

Reply:
I'll update the link to point to this second one better. We should remove then the plugin-workflows repository.